lib/pbench/server/api/__init__.py

```diff
@@ -1,9 +1,16 @@
+"""
+Simple server APIs for redirecting other services call via Pbench server.
+This serves as a middleware for dashboard Elasticseach and GraphQL calls
+as well as Pbench agent move results call.
+"""
```

Comment:
Could we consider a rewording of this a bit?

```
"""Pbench Server API module - Provides middleware for remote clients, either the associated Pbench Dashboard or any number of pbench agent users.
```

Seems like as a module documentation string, we should avoid listing which APIs are provided since they are likely to change.